lib/caesar.rb

```diff
@@ -1,6 +1,44 @@
 class Caesar
-  attr_reader :key
-
+  attr_reader :character_hash
   def initialize
+    @characters = ("a".."z")
+    @character_hash = {}
+  end
+
+  def rotate_characters(rotation)
+    @characters.to_a.rotate(rotation)
+  end
+
+  def cypher(rotation)
```

Comment:
I would have named this a little bit more specifically. For instance, create_cipher, or generate_cipher